lib/active_shipping/errors.rb

```diff
@@ -2,6 +2,9 @@ module ActiveShipping
   class Error < ActiveUtils::ActiveUtilsError
   end
 
+  class RequestError < ActiveShipping::Error
```

Comment:
`ActiveShipping::Error` is an `ActiveUtils::ActiveUtilsError`, which is an alias for `HTTPRequestError`. Not really appropriate, but I thought I'd point it out.